### ISSUE TITLE
Not resolving imports correctly

### ISSUE THREAD

**Issue by Zodiase** [Meteor-Community-Packages/meteor-scss#239]
I'm getting errors that look like:

```Bash
=> Errors prevented startup:                  
   
   While building for web.browser:
   /imports/startup/client/main.scss: Scss compiler error: Undefined variable: "$mdc-typography-styles".
```

Check out https://github.com/Zodiase/Meteor-Tests/tree/mdc-demo--sass-issue for reproduction.

https://github.com/Zodiase/Meteor-Tests/blob/mdc-demo--sass-issue/meteor-app/imports/startup/client/main.scss shows how the dependencies should be resolved and how there shouldn't be any issue following that strategy.

It uses a Meteor package `zodiase:mdc-sass` which is the one I'm working on. All the SASS files are adopted from the official Material Components repo and shouldn't have dependency bugs. The errors I'm having now make the package completely unusable.

**Comment by peshi** [comment]
@fourseven @sebakerckhof 

Having the same issues.
Using: --release 1.4.3.1

package.json
```
    "bootstrap": "^4.0.0-alpha.6",
    "font-awesome": "^4.7.0",
```

main.scss
```
@import "{}/node_modules/bootstrap/scss/bootstrap.scss";
@import "{}/node_modules/font-awesome/scss/font-awesome.scss";
```

Result:
```
   While processing files with fourseven:scss (for target web.browser):
   /client/main.scss: Scss compiler error: Undefined variable: "$fa-font-path".
```

**Comment by minau87** [comment]
Can confirm what @peshi said, exactly the same issue on my end.

**Comment by peshi** [comment]
@minau87 Meanwhile if you, like me need continue your work - you could manually create your own local bootstrap.scss and font-awesome.scss

I know it sounds silly, and it is - but it's working on my end until this is fixed.

Example ../client/stylesheets/bootstrap.scss

And in that file you create a copy of bootstrap's imports like this.
```
@import "{}/node_modules/bootstrap/scss/variables";
... all the other bootstrap imports ...
@import "{}/node_modules/bootstrap/scss/utilities";
```
and ../client/stylesheets/font-awesome.scss
```
@import "{}/node_modules/font-awesome/scss/_variables";
... all the other font-awesome imports ...
@import "{}/node_modules/font-awesome/scss/_screen-reader";

```

Import your local bootstrap.scss and font-awesome.scss in your main.scss file.

**Comment by minau87** [comment]
@peshi Thanks, I'll consider doing that. 
For the moment, I'm just grabbing font-awesome via CDN - I thought that was, until fixed, probably the easiest solution. Again, thanks for the tip!

**Comment by lmaddio** [comment]
Same problem here, I am getting 
Errors prevented startup:                  
   
   `While processing files with fourseven:scss (for target web.browser):`
   `/client/index.scss: Scss compiler error: File to import:`
   `{}/node_modules/font-awesome/scss/font-awesome.scss not found in file:`
   `/home/xxxxx/xxxxxx/{}/client/index.scss`

Doesn't happen with bootstrap don't know why

**Comment by Zodiase** [comment]
@fourseven @sebakerckhof any updates?

**Comment by holmrenser** [comment]
This problem still exists

**Comment by xumoulei** [comment]
Found same issue

**Comment by xumoulei** [comment]
=> Errors prevented startup:

   While processing files with fourseven:scss (for target web.browser):
   /client/scss/core/core.scss: Scss compiler error: Undefined variable: "$gray-200".

   /client/scss/style.scss: Scss compiler error: Undefined variable: "$aside-menu-width".

**Comment by Floriferous** [comment]
Just ran into this as well. No idea why it suddenly started happening.

**Comment by vincro** [comment]
Any further updates on this?

**Comment by derwaldgeist** [comment]
I guess I ran into the same problem with #282. Adding the references manually is a weird workaround, especially you use complex SCSS frameworks. Re-adjusting these references on every update is just too error-prone. Would love to see a fix for this.

**Comment by AncientChild** [comment]
problem still persists!

**Comment by zrisher** [comment]
Also running into this issue with `fourseven:scss@4.10.0`, `bootstrap-scss@4.3.1`, `bootstrap-vue@2.0.0-rc.24` :

```
@import '../../../node_modules/bootstrap-scss/bootstrap';
```

will work fine and include all child imports properly.

```
@import '../../../node_modules/bootstrap-vue/src/index';
```

will fail with this error:

```
While processing files with fourseven:scss (for target web.browser):
   /client/main.scss: Scss compiler error: Error: Undefined variable: "$b-table-busy-opacity".
```

even though the imported file includes a variables file that defines that variable, and if I include the file manually:

```
@import '../../../node_modules/bootstrap-vue/src/_variables.scss';
@import '../../../node_modules/bootstrap-vue/src/index';
```

it compiles without issue.

**Comment by sebakerckhof** [comment]
Fixed per 4.14.1
